### ISSUE TITLE
Create dataset loader for GKLMIP Product Sentiment

### ISSUE THREAD

**Issue by SamuelCahyawijaya** [SEACrowd/seacrowd-datahub#358]
Dataloader name: `gklmip_sentiment/gklmip_sentiment.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?gklmip_sentiment

|     Dataset| gklmip_sentiment |
|-------------|---|
| Description | The GKLMIP Product Sentiment Dataset is a Burmese dataset for sentiment analysis. It was created by crawling comments on an e-commerce website. The sentiment labels range from 1 to 5, with 1 and 2 being negative, 3 and 4 being neutral, and 5 being positive. |
| Subsets | - |
| Languages | mya |
| Tasks | Sentiment Analysis |
| License | Unknown (unknown) |
| Homepage | https://github.com/GKLMIP/Pretrained-Models-For-Myanmar/tree/main |
| HF URL | - |
| Paper URL | https://link.springer.com/chapter/10.1007/978-3-030-92310-5_52 |

**Comment by djanibekov** [comment]
#self-assign

**Comment by djanibekov** [comment]
yes, still in progress
